test/JDBC/expected/SELECT_INTO_TEST-vu-verify.out

```diff
@@ -0,0 +1,304 @@
+
+-- create a new table select_into 1
+create table select_into(select_into_COL int);
+go
+select select_into_COL from select_into;
+go
+~~START~~
+int
+~~END~~
+
+-- Create a new table select_into_REPO with column select_into_COL 
+select select_into_COL into select_into_REPO from select_into;
+go
+-- Indexing over column select_into_COL
+create index IDX_REPRODUCTION on select_into_REPO(select_into_COL);
+go
+-- output is the lowercase name,original_name
+select attname,attoptions from pg_attribute join pg_class on pg_attribute.attrelid=pg_class.oid where pg_class.relname='select_into_repo' and attname like '%select_into%'order by attname asc;
```

Comment:
Have you considered using `information_schema.columns` instead?

Reply:
We want to test both downcased name and original name for the column whereas we will only get the original name from information_schema.columns

Reply:
information_schema.columns returns either original_name or stored name depending upon whether or not the original_name exists.